### PR TITLE
py-bottlenose: new port

### PR DESCRIPTION
**Pull request by catap**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 11.5.2 20G95 x86_64
Xcode 12.5.1 12E507

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-bottlenose/Portfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        lionheart bottlenose 1.1.10
```
Comment (reneeotten) [inline python/py-bottlenose/Portfile:7]: any reason not to take the latest release, or did you meant to type 1.1.18? Actually, that is confusing... PyPI says 1.18p but the latest tag on GitHub is 1.1.10 - can you look into that?

Comment (reneeotten) [reply python/py-bottlenose/Portfile:7]: also, please specify where to get the tarball from when using the github PG. "releases" is preferred if upstream does that, next-best option is "archive". If you change that you might need to update the checksums as well (don't forget to do port clean first).

Comment (catap) [reply python/py-bottlenose/Portfile:7]: @reneeotten PyPi says that the last release is `1.1.8`, not `1.1.18` ;)

Comment (reneeotten) [reply python/py-bottlenose/Portfile:7]: oh well... reading is apparently difficult on the Saturday morning... the point about `github.tarball_from` is still relevant though - correct?

Comment (catap) [reply python/py-bottlenose/Portfile:7]: @reneeotten upstream doesn't serve releases => as you may see I've added `github.tarball_from archive`.

I expect that `port clean` cleans also distfile but it doesn't => I've re-run it with `--all` and here an updated checksums.

**Pull request merged**